background-remover/background-remover.py

```diff
@@ -0,0 +1,55 @@
+import os, glob
+import rembg
+
+def process (input_path: str, output_path: str, session: rembg.bg.BaseSession) -> bool:
+    print(f"making file at {input_path} to {output_path}")
+    if not os.path.exists(input_path):
+        print(f"ERROR: Process failed due to input file does not exist [{input_path}]")
+        return False
+        
+    output_dir_path = os.path.dirname(output_path)
+    if not os.path.isdir(output_dir_path):
+        print(f"LOG: Process creates output directory [{output_dir_path}].")
+        os.makedirs(output_dir_path)
+
+    try:
+        with open(input_path, 'rb') as i:
+            input = i.read()
+
+        output = rembg.remove(input, session=session)
+
+        with open(output_path, 'wb') as o:
+            o.write(output)
+
+    except Exception as error:
+        print(f"ERROR: Process failed due to exception [{error}]")
+        return False
+
+    return True
+
+# Getting files
+in_dir  = f"{os.path.dirname(__file__)}/test/in"
+out_dir = f"{os.path.dirname(__file__)}/test/out"
+
+if __name__ == '__main__':
```

Comment:
focus on creating the simplest solution to the problem, don't add any other logic, right now we only need to process items by one